README.md

```diff
@@ -25,12 +25,12 @@ npm install @preact/signals-react
 	- [`computed(fn)`](#computedfn)
 	- [`effect(fn)`](#effectfn)
 	- [`batch(fn)`](#batchfn)
-- [Preact Integration](#preact-integration)
-	- [Hooks](#hooks)
-	- [Rendering optimizations](#rendering-optimizations)
-		- [Attribute optimization (experimental)](#attribute-optimization-experimental)
-- [React Integration](#react-integration)
-	- [Hooks](#hooks-1)
+- [Preact Integration](./packages/preact/README.md#preact-integration)
+	- [Hooks](./packages/preact/README.md#hooks)
+	- [Rendering optimizations](./packages/preact/README.md#rendering-optimizations)
+		- [Attribute optimization (experimental)](./packages/preact/README.md#attribute-optimization-experimental)
+- [React Integration](./packages/react/README.md#react-integration)
+	- [Hooks](./packages/react/README.md#hooks)
 - [License](#license)
```

Comment:
Is it weird if this links to other docs? It's sorta laid out as a table of contents, but if we link elsewhere, then it stops being the contents of this doc.

Reply:
I think it's fine